docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ysql.md

```diff
@@ -40,24 +40,28 @@ You can restore YugabyteDB universe YSQL data from a backup as follows:
 
 2. If you want to restore a backup from a specific keyspace, click on the backup and use its **Backup Details** page to perform the restore procedure.
 
-3. If you want to restore a full backup, use the **Backups** page to select the backup and click its **... > Restore Entire Backup**, as per the following illustration:<br>
+3. If you want to restore a complete backup, use the **Backups** page to select the backup and click its **... > Restore Entire Backup**, as per the following illustration:<br>
 
     ![Restore backup](/images/yp/restore-entire-backup-ysql.png)
 
-4. Complete the fields of the **Restore Backup** dialog, as follows:
+    Complete the fields of the **Restore Backup** dialog, as follows:
 
     - Select the name of the universe to which you want to restore the backup.
-    
+
     - Optionally and depending on your cloud provider, if the backup was from a universe that has [encryption at rest enabled](../../../security/enable-encryption-at-rest), then select the KMS configuration to use.
-    
-    - Refrain from selecting  **Rename databases in this backup before restoring**.
-    
+
+    - Refrain from selecting **Rename databases in this backup before restoring**.
+
     - Optionally, specify the number of parallel threads that are allowed to run. This can be any number between 1 and 100.
-    
+
     - Click **Restore**.<br>
-    
+
       The restore begins immediately. When finished, a completed **Restore Backup** task appears under **Tasks > Task History**.
-    
+
+4. If your backup includes incremental backups, you can either restore the complete backup, as per step 3, or restore a part of an incremental backup chain by selecting an increment from the list in the **Backup Details** view and clicking its **Restore to this point**.
+
+    During the restore, only successful incremental backups are stored and any failed backups are discarded.
```

Comment:
During the restore, only successful full and incremental backups are used and any failed backups are discarded.

Reply:
Changed